index.js

```diff
@@ -51,29 +51,30 @@
           i       = directions.length,
           coords,
           scrollTop  = document.documentElement.scrollTop || document.body.scrollTop,
-          scrollLeft = document.documentElement.scrollLeft || document.body.scrollLeft
+          scrollLeft = document.documentElement.scrollLeft || document.body.scrollLeft;
 
       nodel.html(content)
-        .style({ opacity: 1, 'pointer-events': 'all' })
+        .style('opacity', 1).style('pointer-events', 'all');
```

Comment:
Yes. It has been moved to a plugin which is not included by default

From https://github.com/d3/d3-selection/blob/master/README.md

> For the sake of parsimony, the multi-value methods—where you pass an object to set multiple attributes, styles or properties simultaneously—have been extracted to d3-selection-multi and are no longer part of the default bundle.